core/src/main/java/feast/core/service/JobCoordinatorService.java

```diff
@@ -88,32 +92,9 @@ public JobCoordinatorService(
   @Scheduled(fixedDelayString = "${feast.jobs.polling_interval_milliseconds}")
   public void Poll() throws InvalidProtocolBufferException {
     log.info("Polling for new jobs...");
-    @Positive long updateTimeout = jobProperties.getJobUpdateTimeoutSeconds();
-    List<JobUpdateTask> jobUpdateTasks = new ArrayList<>();
-    ListStoresResponse listStoresResponse = specService.listStores(Filter.newBuilder().build());
+    Map<Source, Store> sourceStoreMap = getSourceToStoreMapping();
```

Comment:
Why it's a map? Is it 1<->1 relation?
One source can be used to populate many stores
I think it's rather list of Pairs

Reply:
Corrected